apps/participant-portal/src/pages/TeamSetup.css

```
@@ -0,0 +1,469 @@
+  --ink: #1d1d1f;
+  --ink-2: #424245;
+  --ink-3: #6e6e73;
+  --ink-4: #86868b;
+  --line: #d2d2d7;
+  --line-soft: #e8e8ed;
+  --paper: #ffffff;
+  --paper-2: #fbfbfd;
+  --paper-3: #f5f5f7;
+  --battle: #ff453a;
+  --ok: #30b766;
+  --font-sans: "Inter", "Noto Sans JP", -apple-system, BlinkMacSystemFont, system-ui, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)